### PR TITLE
WIP - issue 494 added npm install

### PR DESCRIPTION
**Pull request by lavernw1**
This is a WIP please review. Updated .adoc and readme .property files for NodeJS booster.
@rhoads-zach @tradej  please review.  @tradej I Inadvertently did push before deleting the .xmls generated running the scripts to build the guides. Removed the untracked files after git push with delete and git --amend and push . Still shows up in the pull request.  For this change, should we update all the mission .adoc files instead of just the health check and rest http?

### CODE REVIEW

**Comment by rhoads-zach** [comment]
@lavernw1 
* please remove the master.xml files
* merge any new content in scripts/.gitignore into .gitignore and remove scripts/.gitignore.

Other than that, content looks good.

**Comment by rhoads-zach** [comment]
LGTM, thank you @lavernw1 

Merging.

**Comment by rhoads-zach** [comment]
merged;:d4ee7b1b452f21952ceb591a1f227df147506e11

**Comment by Ladicek** [comment]
> These commands use the Nodeshift `npm` module to install {NodeJS}

That's not true. `npm install` doesn't install Node.js -- it installs npm packages, or in our case, package dependencies. See https://docs.npmjs.com/cli/install.

I'd rephrase this

> These commands use the Nodeshift `npm` module to install {NodeJS} and launch the S2I process on {OpenShiftLocal} and start the pod.

as this: These commands first install any possibly missing package dependencies and then, using the Nodeshift package, launch the S2I process on {OpenShiftLocal} and start the pod.
